packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -132,7 +132,7 @@ export class DocumentDeltaConnection
 			logger.sendErrorEvent(
 				{
 					eventName: "DeltaConnection:EventException",
-					name,
+					name: name as string,
```

Comment:
`EventEmitterEventType` explicitly permits `name` to be `string | symbol`, but `sendErrorEvent` definitely wants a string.

Probably the best option is to restrict `EventEmitterEventType` to just `string` for simplicity, but I don't do that in this change as it would be a public API change.

Reply:
Not sure what this is accomplishing. At runtime if this was a Symbol it'll still be one, and I don't understand how the cast would help TS at development time.

Reply:
The typing changes suddenly made TS realize this should fail type checking, so the cast allows it to build again.

If a Symbol were being passed it would throw later down the line (e.g. in something like `convertToBasePropertyTypeUntagged`) since there's not really a reasonable way to convert a Symbol to something that can be serialized/logged.  So this cast just makes that assumption explicit; that although `name` technically could be a Symbol here, we're really assuming it's not if we're willing to pass it into `sendErrorEvent`.

My comment above is basically saying "why risk it, it would make more sense to just ban Symbol as eventName throughout the repo and force everyone to use strings, no one like Symbols anyway".  Then this cast could go away.  But doing so would technically be a breaking change.